ravenpy/models/base.py

```diff
@@ -544,55 +507,11 @@ def _get_output(self, pattern, path):
         files = list(path.rglob(pattern))
 
         if len(files) == 0:
-            if not (
-                isinstance(self.config.rvi, RVI) and self.config.rvi.suppress_output
-            ):
+            if not self.config.rvi.suppress_output:
```

Comment:
What happens if the user does not use an emulator, but existing RV files (not templated) ?

Reply:
It wouldn't work.. it would require RV parsing, which is implemented only for RVC currently.

Reply:
That's a problem.

Reply:
ça marchait pas plus avant non ?

Reply:
Je vois ce que tu veux dire. Effectivement, ça ne changeait rien. 
Le problème était que `self.config.rvi` n'avait pas d'attribut `suppress_output`, et donc ça déclenchait une erreur. Ici j'imagine qu'il y a une valeur par défaut qui s'applique dans le cas des émulateurs et des fichiers RV tous remplis.